pom.xml

```diff
@@ -79,6 +79,7 @@
         <ddf-branding>DDF</ddf-branding>
         <ddf-branding-lowercase>ddf</ddf-branding-lowercase>
         <ddf-branding-expanded>Distributed Data Framework</ddf-branding-expanded>
+        <cal-branding-expanded>Codice Alliance</cal-branding-expanded>
```

Comment:
Nothing formal. I guess that's a branding decision ultimately. I'm not 100% sure it's necessary to say "Codice Alliance" every time, but that's how it was in this intro section. Afterward, it's always just "Alliance"